docs/sources/reference/components/prometheus/prometheus.exporter.mssql.md

```diff
@@ -27,6 +27,7 @@ You can use the following arguments with `prometheus.exporter.mssql`:
 | Name                   | Type       | Description                                                         | Default | Required |
 | ---------------------- | ---------- | ------------------------------------------------------------------- | ------- | -------- |
 | `connection_string`    | `secret`   | The connection string used to connect to an Microsoft SQL Server.   |         | yes      |
+| `connection_name`      | `string`   | The connection string used to connect to an Microsoft SQL Server.   |         | no       |
```

Comment:
This description looks to be the same as `connection_string`, and if it is *not* required, you should specify what the default value is.

Reply:
Yes, "" would be correct.